src/main/java/com/theelixrlabs/healthcare/model/Doctor.java

```diff
@@ -0,0 +1,20 @@
+package com.theelixrlabs.healthcare.model;
+
+import com.theelixrlabs.healthcare.constants.DoctorConstants;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.UUID;
+
+@Data
+@Document(collection = DoctorConstants.DOCTOR_COLLECTION_NAME)
+public class Doctor {
```

Comment:
You can name the classes as per their usage
DoctorsModel ar DoctorsEntity

Also add some class level comments

Reply:
the name changed to "DoctorsEntity". and aslo added some class level comments.